core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -1347,11 +752,11 @@ impl<R: WriteableRocks> ExecutedScenarioStore for StateManagerDatabase<R> {
 
 pub struct RocksDBCommittedTransactionBundleIterator<'r> {
     state_version: StateVersion,
-    txns_iter: Box<dyn Iterator<Item = (StateVersion, RawLedgerTransaction)> + 'r>,
-    ledger_receipts_iter: Box<dyn Iterator<Item = (StateVersion, LedgerTransactionReceipt)> + 'r>,
-    local_executions_iter: Box<dyn Iterator<Item = (StateVersion, LocalTransactionExecution)> + 'r>,
+    txns_iter: Box<dyn Iterator<Item=(StateVersion, RawLedgerTransaction)> + 'r>,
```

Comment:
It looks like some weird non-standard formatter ran on this file, could we revert these changes?

I'd imagine the standard formatter should revert them, but maybe the changes aren't in the standard formatter's preference.

Still, we use spaces around our `=` and cargo format automatically indents the `for` when it puts it on a new line, so lets keep that.

Reply:
Fixed.

Reply:
Reformatted files with Cargo fmt.